### PR TITLE
adding selenium-tests

### PR DESCRIPTION
**Pull request by Jdure**

### CODE REVIEW

**Comment by jaysonmc** [comment]
Great work! 

I imagine 'localhost:8888' should be passed in as an env variable, no? @e-wu can confirm. Otherwise it'll only work locally not part of the deployment magic

File: testing/lib/basePage.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        .build();
+    } else {
+        this.driver = new webdriver.Builder().withCapabilities(webdriver.Capabilities.chrome()).build();
+    }
+
+    this.baseUrl = function() {
+        if (process.env.SECURITY_MODE === 'true') {
+            return 'http://odoo:8069';
+        } else {
+            return 'http://localhost:8069';
```
Comment (e-wu) [inline testing/lib/basePage.js:33]: As per Jayson's suggestion, can you make localhost:8069 an environment variable.

File: testing/lib/basePage.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        .withCapabilities(webdriver.Capabilities.chrome())
+        .setProxy(proxy.manual({http: 'localhost:8888'},{https: 'localhost:8888'}))
+        .build();
+    } else {
+        this.driver = new webdriver.Builder().withCapabilities(webdriver.Capabilities.chrome()).build();
+    }
+
+    this.baseUrl = function() {
+        if (process.env.SECURITY_MODE === 'true') {
+            return 'http://odoo:8069';
```
Comment (e-wu) [inline testing/lib/basePage.js:31]: As per Jayson's suggestion, can you make localhost:8069 an environment variable.

File: testing/lib/basePage.js (hunk 1 above)
Comment (Jdure) [reply testing/lib/basePage.js]: Done

File: testing/lib/basePage.js (hunk 2 above)
Comment (Jdure) [reply testing/lib/basePage.js]: Done

**Comment by Jdure** [comment]
> Sorry for the confusion, meant the entire http://localhost:8869 should be an environment variable like process.env.SITE_URL

Okay no worries. It's done

**Pull request merged**